### DISCUSSION TITLE
Pagination with DaisyUI

### DISCUSSION THREAD

**Discussion by abkrim** [livewire/livewire#8638] [1. Help]
I want to use pagination on my site that uses DaisyUI.

When trying to do so I have observed a problem that causes me to not be able to use the personalization.

1. publish `livewire:publish --config`
2. Check the 4 `vendor/livewire` files on pagination
3. try using one of the two tailwind ones to check my problem

`{{ $articles->links() }}`
Perfect

Now we change to `{{ $articles->links('livewire::simple-tailwind') }}` one of the official published templates.

And perfect. It looked good, but the reactivity didn't work. That is, I couldn't give next or previous.

What's more, I lose the reactivity of the page, where I have a drop-down menu to choose the language.

Anyone know?

**Comment by joshhanley** [comment]
@abkrim if you want simple layout, you shouldn't pass the view into links, instead you should use the simple paginator as documented here https://livewire.laravel.com/docs/pagination#using-the-simple-theme

Hope this helps!

**Comment by Smeedsajjad** [comment]
I have try this, I got the buttons on the browser. But the problem is the reactivity still not working.
Anyone know?
